samples/sensor/bmm150/src/main.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <misc/printk.h>
+#include <sensor.h>
+#include <stdio.h>
+
+void do_main(struct device *dev)
+{
+	int ret;
+	struct sensor_value x, y, z;
+
+	while(1) {
+		ret = sensor_sample_fetch(dev);
+		if(ret) {
+			printk("sensor_sample_fetch failed ret %d\n",ret);
+			return;
+		}
+
+		ret = sensor_channel_get(dev, SENSOR_CHAN_MAGN_X, &x);
+		ret = sensor_channel_get(dev, SENSOR_CHAN_MAGN_Y, &y);
+		ret = sensor_channel_get(dev, SENSOR_CHAN_MAGN_Z, &z);
+		
+		printf("( x y z ) = ( %f  %f  %f )\n",
+                       sensor_value_to_double(&x),
```

Comment:
one space too many for the alignment (unless it's github's view which makes it weird). These should be aligned right below the "